### ISSUE TITLE
Wrong "is instance" for preprint 

### ISSUE THREAD

**Issue by fnielsen** [citation-js/citation-js#136]
For instance, https://www.medrxiv.org/content/10.1101/2021.08.15.21262067 results in manuscript (document written by hand).  https://www.wikidata.org/w/index.php?title=Q109622851&oldid=1529788551

I would have expected that is should be something else, e.g., preprint.

**Comment by fnielsen** [comment]
I forgot to check whether this is already fixed. We are seeing the error in Scholia https://github.com/WDscholia/scholia/issues/1704

**Comment by fnielsen** [comment]
Current master:
```bash
$ citation-js -t https://doi.org/10.1101/2021.07.28.21261159 | python -m json.tool | grep '"type"'
        "type": "manuscript",
                "type": "@doi/api",
                "type": "@csl/list+object"
```

**Comment by larsgw** [comment]
The CrossRef API is (still!) returning [invalid CSL-JSON](https://github.com/CrossRef/rest-api-doc/issues/187). They are using the type `"posted-content"` (should be `"article"`) with the `subtype` field (undocumented, they should use `genre`) set to `"preprint"`. At the time it was a bit unclear what `"posted-content"` should be matched to, and I used [this mapping](https://github.com/greenelab/deep-review/blob/b064dc298a55249e7091fe8197dce401d8cf6c12/build/citations.py#L108-L1130). I'm still not sure if I can always map `"posted-content"` to `"article"`, who knows what other `subtype`s there might be.

**Comment by fnielsen** [comment]
Would this be better: https://www.wikidata.org/wiki/Q2376293. It is also call a "manuscript".

**Comment by larsgw** [comment]
The [CSL documentation](https://docs.citationstyles.org/en/v1.1/specification.html#appendix-iii-types) says this about `manuscript`

> An unpublished manuscript; Use for both modern unpublished works and classical manuscripts; For working papers, preprints, and similar works posted to a repository, use `article`.

So I think the issue lies more with the CrossRef data. I can make a workaround in Citation.js, though I'd prefer CrossRef fixed their API of course.

By the way, the mappings to Wikidata are based on the [exact match (P2888)](https://www.wikidata.org/wiki/Property:P2888) statements ([query](https://w.wiki/4Q$f))

**Comment by fnielsen** [comment]
In Scholia we got [manuscript: 'Q87167'](https://github.com/WDscholia/scholia/blob/37d90f0288448f02c38c94d24a58a6105fae05fd/scholia/app/static/js/citation.js#L7543) 

This should be `manuscript: 'Q2376293',`.

I am not immediately able to see where that line comes from in the citation-js code

**Comment by larsgw** [comment]
This seems to be fixed in [v0.5.3](https://github.com/citation-js/citation-js/releases/tag/v0.5.3), not sure why I kept it open.
